main.cpp

```diff
@@ -92,6 +92,59 @@ good to go!
  */
 
 #include <iostream>
+
+struct FloatType
+{
+public:
+    float add(float lhs, float rhs) { return lhs + rhs; }
```

Comment:
`struct` is `public` by default.
this access specifier is unnecessary
- [ ] revise